torch/jit/supported_ops.py

```diff
@@ -249,12 +261,12 @@ def _get_global_builtins():
             table_row = '":any:`{}`", "{}"'.format(fn, schemaless_op_explanations[fn])
             schemaless_ops.append(table_row)
 
-    schematized_ops = '\n'.join(schematized_ops)
-    schemaless_ops = '\n'.join(schemaless_ops)
-    magic_methods_rows = '\n'.join(magic_methods_rows)
-    schematized_ops = textwrap.indent(schematized_ops, '\t')
-    schemaless_ops = textwrap.indent(schemaless_ops, '\t')
-    magic_methods_rows = textwrap.indent(magic_methods_rows, '\t')
+    schematized_ops_str = '\n'.join(schematized_ops)
+    schemaless_ops_str = '\n'.join(schemaless_ops)
+    magic_methods_rows_str = '\n'.join(magic_methods_rows)
+    schematized_ops_str = textwrap.indent(schematized_ops_str, '\t')
+    schemaless_ops_str = textwrap.indent(schemaless_ops_str, '\t')
+    magic_methods_rows_str = textwrap.indent(magic_methods_rows_str, '\t')
```

Comment:
You renamed these variables, but didn't use the new names further down. That explains the doc build failure in CI.